src/components/Reports/LocationSection.tsx

```diff
@@ -128,6 +234,78 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
           </Grid>
         </>
       )}
+      {isPlantingSite && (
```

Comment:
I can separate them, but they are pretty much the same, that's why I added all them here, not sure what is better

Reply:
> I can separate them, but they are pretty much the same, that's why I added all them here, not sure what is better

we can leave it be for now and see what Charlie has to say.. we can always refactor later.. btw the tests are failing

Reply:
Hmm, yeah it does seem like a lot of conditional rendering. You're right that there are quite a few common bits, though. I suppose I'm with Karthik: I think it would be fine to keep this as is and refactor it later if we decide this is too difficult to modify.

Reply:
Sounds good!